src/main/java/TaskManager.java

```diff
@@ -0,0 +1,128 @@
+public class TaskManager {
+    public static final int MAX_TASK = 100;
+    private static Task[] tasks;
+    private static int taskCount;
+
+    public TaskManager() {
+        tasks = new Task[MAX_TASK];
+        taskCount = 0;
+    }
+
+    public void addTask(String input) {
+        int taskLength = tasks.length;
+
+        if (taskCount >= taskLength) {
+            System.out.println("""
+                ____________________________________________________________
+                You're too busy, go get some rest...ZZZ
+                ____________________________________________________________
+                """);
+            return;
+        }
+
+        if (input.startsWith("event ")) {
+            String[] parts = input.substring(6).split(" /from ");
+            if (parts.length != 2) {
+                System.out.println("""
+                    ____________________________________________________________
+                    Invalid event format. Please use: event description /from time /to time
+                    ____________________________________________________________
+                    """);
+                return;

```

Comment:
Great that your function terminates if the input is invalid, without the program crashing 👍🏼